api/migrations/0002_data_migration.py

```diff
@@ -1,31 +1,19 @@
 import datetime
 
 from django.db import migrations
-from django.db.backends.postgresql.schema import DatabaseSchemaEditor
-from django.db.migrations.state import StateApps
 
+from api.models.age_ranges import AgeRangeType
 from api.models.constants import (
     STUDENT_AGE_RANGES,
     STUDENT_AGE_RANGES_FOR_MATCHING,
     STUDENT_AGE_RANGES_FOR_TEACHER,
 )
-from api.models.age_ranges import AgeRangeType
+from api.models.helpers import DataMigrationMaster
 
 APP_NAME = "api"
 
 
-class PrePopulationMaster:
-    """Class-based variation of functions for pre-populating the database with fixed data.
-    See Django docs: https://docs.djangoproject.com/en/stable/topics/migrations/#data-migrations
-    """
-
-    def __init__(self, apps: StateApps, schema_editor: DatabaseSchemaEditor):
-        self.apps = apps
-        self.schema_editor = schema_editor
-        # Adding __call__ to a class won't work because RunPython can't create an instance and then
-        # run it.  So we're running main() right after instantiating, which effectively makes
-        # PrePopulationMaster into a callable with instance attributes.
-        self.main()
+class InitialDataPopulationMaster(DataMigrationMaster):
```

Comment:
```suggestion
class InitialDataPopulator(DataMigrationMaker):
```
see comment in`helpers.py`

Reply:
Here I wanted to be consistent with DataMigrationMaker base class name, so in my mind it should be called InitialDataMaker and FakeDataMaker.

But renamed it as you suggested :)

Reply:
You know what, you're right. I think it was my mistake to call it `DataMigrationMaker` because `makemigrations` actually **creates** migration files (whereas we create the data migration module by hand), and then the migrations are applied when `migrate` is run.

On the other hand, I didn't like the `...Master` thingy because it sounded too vague. So how about calling the abstract base class `Populator` or something like that?

Reply:
renamed it to Populator.

hardest thing in programming is naming :D